app/core/model/src/main/java/io/syndesis/model/metrics/MetricsSummary.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.model.metrics;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.Optional;
+
+import org.immutables.value.Value;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import io.syndesis.model.Kind;
+import io.syndesis.model.WithId;
+
+@Value.Immutable
+@JsonDeserialize(builder = MetricsSummary.Builder.class)
+@SuppressWarnings("immutables")
+public interface MetricsSummary extends WithId<MetricsSummary>, Serializable{
```

Comment:
If this is just a response DTO, why does it have an ID? Doing so means that non-jsondb implementations, i.e. Prometheus, will have to insert a dummy ID. I would prefer to have a response type MetricsSummary that is expose from the API and the ID be only stored in jsondb, not returned via the API. This is tying the API to the implementation which is not a good idea.

Reply:
I'm not sure what the point of the ID is in the returned integration specified metrics is? If we dropped that, we could just use the same response type for aggregated metrics.

Reply:
@jimmidyson, The id is the integrationId, you really want to remove it?